Marksman/Cst.fs

```diff
@@ -147,6 +149,7 @@ type Element =
     | WL of Node<WikiLink>
     | ML of Node<MdLink>
     | MLD of Node<MdLinkDef>
+    | YML of Node<YamlContent>
```

Comment:
Potentially, could use TextNode here? YamlContent type alias doesn't seem to add that much at the moment.

Reply:
still relevant